### PR TITLE
grafana 9.1.8

### PR DESCRIPTION
**Pull request by ying-jeanne**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/grafana.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Grafana < Formula
   desc "Gorgeous metric visualizations and dashboards for timeseries databases"
   homepage "https://grafana.com"
-  url "https://github.com/grafana/grafana/archive/v9.1.7.tar.gz"
-  sha256 "dac3d154901a9d5e0d1a21f3b3a5b06ea62e86aa03c4e2fa1fe38257887f3439"
+  url "https://github.com/grafana/grafana/archive/v9.1.8.tar.gz"
```
Comment (chenrui333) [inline Formula/grafana.rb:4]: why not 9.2.0?

Comment (ying-jeanne) [reply Formula/grafana.rb:4]: this is automatically created by release tool, we were releasing both grafana v9.1.8 and v9.2.0. should we keep the latest version instead?

Comment (SMillerDev) [reply Formula/grafana.rb:4]: Yes, it should always be the latest version

Comment (ying-jeanne) [reply Formula/grafana.rb:4]: if it is, then the URL should point to v9.2.0 you are right, and we need to improve our release tool as well.

**Pull request closed without merging**